### ISSUE TITLE
task01: simulate

### ISSUE THREAD

**Issue by Revilllo7** [Revilllo7/computational-intelligence-class#46]
Do the following:

- Create the script/function `forward propagation` for a neural network that has the same weights as the graphic and a sigmoidal activation function on the hidden layers. The way which you program the function is up to you, the important part is understanding what's going inside and how the forward propagation algorithm works.
- Test the functionality on entry `x1=0.6, x2=0.1` Test if it returns around 0.234. If so, it's written correctly
- Add backpropagation, gradient decrease, MSE. As written in the following files
$L = \frac{1}{2}*(\upsilon - y)^2$ and the learning speed $eta = 0.1$
- Test how the weights will be adjusted after the learning algorithm. Test if they are the same as in the script [image given]
